src/UI/Utils/Element.elm

```diff
@@ -125,6 +126,59 @@ colorTransition time =
         |> List.map tuplesToStyles
 
 
+type alias Transition msg =
```

Comment:
I would make create a specific module for animation as it is a closed "realm"

Reply:
I agree but I think that it can be laid out better once it's included officially in the design system, which Sufiyan already said to be one of the goals after standardizing